### DISCUSSION TITLE
[impossible to downagrade] I can't downgrade from pro to free

### DISCUSSION THREAD

**Discussion by maxbellec** [community/community#45998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

From the billing page, when I try downgrading to Free, using this modal :
![image](https://user-images.githubusercontent.com/7253902/215992333-9b7e105c-838d-4cc5-b8db-49697be42bd4.png)

I get a wrong confirmation message and I'm still on the pro plan.
![image](https://user-images.githubusercontent.com/7253902/215992533-dc5629c9-829a-468f-94be-c6f9cba24f77.png)

BTW I NEVER signed up for the pro plan, it doesn't bother me too too much to pay for one month, but I'm not paying forever...

How should I proceed to downgrade?

**Comment by radar-blip** [comment]
Interesting. You could try [Support](https://support.github.com/contact/account), they should be able to help.

**Comment by jge162** [comment]
Hello @maxbellec please use this link and submit a ticket for billing section -> [GitHub support](https://support.github.com/request/account?tags=hubberfy_account)

**Reply by maxbellec** [reply]
Thank you, I submitted a ticket.

**Comment by Anmol-Baranwal** [comment]
If you published a GitHub Pages site in a private repository and added a custom domain, remove or update your DNS records before downgrading from GitHub Pro to GitHub Free, to avoid the risk of a domain takeover.

Maybe you can try logging again and downgrade from the Billing and plans section although I am not sure if this would solve the issue.

**Reply by maxbellec** [reply]
Thank you for the tip, I have removed Github pages but I had no custom domain. Good idea, I tried logging out and in again, it didn't solve the issue sadly.

**Comment by ettaboyle** [comment]
Welcome to the GitHub Community, [maxbellec](https://github.com/maxbellec), we're happy you're here! 

I checked out the thread and see that you contacted support. That's the best route as they are the team that can help with account and billing questions. I wanted to let you know I re-categorized this post as its not related to the `GitHub Profile` and is more relevant as a `General` topic

No action is needed - just wanted to close the loop on that. I hope you're able to find a resolution!
